builder/relinker/relinker.py

```
@@ -0,0 +1,384 @@
+            secs.append('.text.%s'%(l, ))
+        else:
+            secs.append(l)
+    return secs
+
+class filter_c:
+    def __init__(self, file):
+        with open(file, 'r', encoding='utf-8') as f:
+            lines = f.read().splitlines()
+        self.libs_desc = ''
+        self.libs = ''
+        for l in lines:
```

Ruff (0.15.7)

[error] Ambiguous variable name: `l`

(E741)